client/hosting-overview/components/plan-card.tsx

```diff
@@ -0,0 +1,119 @@
+import { useSelector } from 'react-redux';
+import QuerySitePlans from 'calypso/components/data/query-site-plans';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import useCheckPlanAvailabilityForPurchase from 'calypso/my-sites/plans-features-main/hooks/use-check-plan-availability-for-purchase';
+import { getCurrentPlan } from 'calypso/state/sites/plans/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+const PlanCard: FC = () => {
+	const moment = useLocalizedMoment();
+	const site = useSelector( getSelectedSite );
+	const sitePlan = site?.plan;
+	const planName = sitePlan?.product_name_short ?? '';
+	const planSlug = ( sitePlan?.product_slug || '' ) as PlanSlug;
+	const plan = useSelector( ( state ) => getCurrentPlan( state, site?.ID ) );
+	const pricing = usePricingMetaForGridPlans( {
+		coupon: undefined,
+		planSlugs: [ planSlug ],
+		selectedSiteId: site?.ID,
+		storageAddOns: null,
+		useCheckPlanAvailabilityForPurchase,
+	} );
+	const translate = useTranslate();
+
+	// TODO: Replace with real data using <PlanStorage /> component
+	const usedGigabytes = 47;
+	const availableUnitAmount = 50;
+
+	return (
+		<>
+			<QuerySitePlans siteId={ site?.ID } />
+			<Card className="hosting-overview__card">
+				<div>
+					<div className="hosting-overview__plan-card-header">
+						<h3 className="hosting-overview__plan-card-title">{ planName }</h3>
+
+						<Button
+							className="hosting-overview__link-button"
+							plain
+							href={ `/plans/${ site?.slug }` }
+						>
+							{ translate( 'Manage plan' ) }
+						</Button>
+					</div>
+					<PlanPrice
+						className="hosting-overview__plan-price"
+						currencyCode={ plan?.currencyCode }
+						displayPerMonthNotation
+						isSmallestUnit
+						rawPrice={ pricing?.[ planSlug ].originalPrice.monthly }
+					/>
+					<div className="hosting-overview__plan-info">
+						{ translate( '{{span}}%(rawPrice)s{{/span}} billed annually.', {
+							args: {
+								rawPrice: formatCurrency(
+									pricing?.[ planSlug ].originalPrice.full ?? 0,
+									plan?.currencyCode ?? '',
+									{
+										stripZeros: true,
+										isSmallestUnit: true,
+									}
+								),
+							},
+							components: {
+								span: <span />,
+							},
+						} ) }
+					</div>
+					<div className="hosting-overview__plan-info">
+						{ translate( 'Expires on %s.', { args: moment( plan?.expiryDate ).format( 'LL' ) } ) }
+					</div>
+					<div className="hosting-overview__plan-storage">
+						<div className="hosting-overview__plan-storage-title-wrapper">
+							<div className="hosting-overview__plan-storage-title">
+								<Icon icon={ cloud } />
+								{ translate( 'STORAGE' ) }
+							</div>
+							<span>
+								{ translate(
+									'Using {{usedStorage}}%(usedGigabytes).1fGB{{/usedStorage}} of %(availableUnitAmount)dGB',
+									'Using {{usedStorage}}%(usedGigabytes).1fGB{{/usedStorage}} of %(availableUnitAmount)dGB',
```

Comment:
The rest of Calypso seems to have a space between the digit and GB, which seems cleaner to read. 

**Here:**
<img width="191" alt="Screenshot 2024-04-13 at 20 29 29" src="https://github.com/Automattic/wp-calypso/assets/43215253/bab0f786-33b2-4447-8f85-45d13db89673">

**Elsewhere:**
<img width="126" alt="Screenshot 2024-04-13 at 20 29 32" src="https://github.com/Automattic/wp-calypso/assets/43215253/d6a21fa7-594b-45cc-b27a-51bc04d9361a">
<img width="159" alt="Screenshot 2024-04-13 at 20 29 39" src="https://github.com/Automattic/wp-calypso/assets/43215253/f867db73-276a-4af6-902e-e219f1a5e107">

Reply:
Thanks, addressed on https://github.com/Automattic/wp-calypso/pull/89476/commits/21e2a4ceace5a8bc950a50ed25c6d018b30b7330.